poincare/src/symbol.cpp

```diff
@@ -107,6 +107,15 @@ Expression SymbolNode::shallowReplaceReplaceableSymbols(Context * context) {
   return Symbol(this).shallowReplaceReplaceableSymbols(context);
 }
 
+ExpressionNode::LayoutShape SymbolNode::leftLayoutShape() const {
+  UTF8Decoder decoder(m_name);
+  CodePoint codePoint = decoder.nextCodePoint();
+  if (codePoint == UCodePointNull || decoder.nextCodePoint() == UCodePointNull) {
```

Comment:
Shouldn't it be : 
assert(codePoint != UCodePointNull);
if (decoder.nextCodePoint() == UCodePointNull) {
?
(+ take care of unused variable warning)